package.json

```diff
@@ -18,12 +18,14 @@
     "next": "14.0.3",
     "react": "^18",
     "react-dom": "^18",
+    "react-icons": "^5.0.1",
     "react-scroll": "^1.9.0"
   },
   "devDependencies": {
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "@types/react-scroll": "^1.8.10",
```

Comment:
you can explain why you added these libraries, what it solves etc.. if it isn't already explained in the ticket or the PR description. 

Generally, the PR description should have big picture explanations and these internal comments are more specific